src/dataDisplay/table.jsx

```diff
@@ -1,21 +1,350 @@
+        }
+        this.setStickyColumnPositions();
+    }
+
+    setStickyCellsStylesOnScroll({ scrollLeft }) {
+        const {
+            basic,
+            definition,
+            fullWidth,
+        } = this.props;
+        const stickyCellsFirstOfRow = this.tableRef.querySelectorAll(`.${STICKY_CELL_FIRST_OF_ROW_CLASS}`);
+        const stickyCellBorderLeftStyle = scrollLeft > 0 && !basic ? STICKY_CELL_BORDER_STYLE : '';
+        for (let i = 0; i < stickyCellsFirstOfRow.length; i += 1) {
+            stickyCellsFirstOfRow[i].style.borderLeft = stickyCellBorderLeftStyle;
+            const isInitalDefinitionCell = definition && !fullWidth && i === 0;
+            if (isInitalDefinitionCell) {
+                stickyCellsFirstOfRow[i].style.borderTop = backgroundColor.default;
+                stickyCellsFirstOfRow[i].style.boxShadow = '';
+                stickyCellsFirstOfRow[i].style.borderLeft = '';
+                stickyCellsFirstOfRow[i].style.borderRight = STICKY_CELL_BORDER_STYLE;
+            }
+        }
+    }
+
+    parseChildren(children) {
+        const {
+            stickyColumnCount,
+            fixed,
+            singleLine,
+            resizableColumnWidthPercentage,
+        } = this.props;
+        const {
+            adjacentStickyColumnsTotalWidth,
+            stickyTableContainerWidth,
+        } = this.state;
+        const shouldHandleStickyBehavior = !fixed && !singleLine;
+        const parsedChildren = React.Children.map(children, (child) => {
+            if (child.type.name === TABLE_HEADER_TYPE) {
+                const tableHeaderRow = child.props.children;
+                const headerCells = tableHeaderRow.props.children;
+                const parsedHeaderCells = React.Children.map(
+                    headerCells,
+                    (headerCell, index) => {
+                        const isResizable = shouldHandleStickyBehavior &&
+                            (
+                                index === stickyColumnCount - 1 &&
+                                index !== headerCells.length - 1
+                            );
+                        return {
+                            ...headerCell,
+                            props: {
+                                ...headerCell.props,
+                                isResizable,
+                                ...(isResizable && {
+                                    adjacentStickyColumnsTotalWidth,
+                                    stickyColumnCount,
+                                    stickyTableContainerWidth,
+                                    shouldResetWhiteSpaceStyle: shouldHandleStickyBehavior,
+                                    forceTableUpdate: this.forceTableUpdate,
+                                    resizableColumnWidthPercentage,
+                                    // eslint-disable-next-line max-len
+                                    setResizableCellsWordWrapping: this.setResizableCellsWordWrapping,
```

Comment:
Can you please remove the eslint disable line and fix the code.

Reply:
done!